modules/csg/csg.cpp

```diff
@@ -1079,14 +1081,30 @@ void CSGBrushOperation::Build2DFaces::_find_edge_intersections(const Vector2 p_s
 
 		// Check each edge.
 		for (int face_edge_idx = 0; face_edge_idx < 3; ++face_edge_idx) {
-			Vector2 edge_points[2] = {
+			Vector<Vector2> edge_points_and_uvs = {
```

Comment:
Try LocalVector

Reply:
LocalVector doesn't work here as the .find(), both for LocalVector (core/templates/local_vector.h(251)) and std Vector, don't seem able to parse LocalVector as a type for equality checking:
```
error C2678: binary '==': no operator found which takes a left-hand operand of type 'const T' (or there is no acceptable conversion)
        with
        [
            T=LocalVector<Vector2,uint32_t,false,false>
        ]
```
I'm a little over my head here trying to parse what would need to change for this to be allowed.